Assets/Scripts/Extensions/RectTransformExtensions.cs

```diff
@@ -0,0 +1,26 @@
+﻿using UnityEngine;
+
+public static class RectTransformExtensions
+{
+    public static void SetAnchors(this RectTransform self, System.ValueTuple<Vector2, Vector2> anchors)
+    {
+        self.anchorMin = anchors.Item1;
+        self.anchorMax = anchors.Item2;
+    }
+
+    public static void Stretch(this RectTransform self)
+    {
+        self.anchorMin = Vector2.zero;
+        self.anchorMax = Vector2.one;
+    }
+
+    public static void Reset(this RectTransform self)
```

Comment:
The rect transform position/scale is always fucked when you create them at runtime. So this just corrects em. Might come in handy later anyway.

Reply:
Good idea